service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseValidatorServiceTest.java

```diff
@@ -182,8 +192,26 @@ void shouldNotReturnErrorsIfMandatoryFieldsHaveBeenCompleted() {
             .orders(Orders.builder()
                 .orderType(ImmutableList.of(OrderType.EMERGENCY_PROTECTION_ORDER))
                 .build())
-            .children1(initChildren())
-            .applicants(initApplicants())
+            .children1(createPopulatedChildren())
```

Comment:
I would maybe suggest extracting this to private methods, 30 lines of setup isn't cool